src/guide/backporting.md

```diff
@@ -7,9 +7,9 @@
 * [Skara documentation on backports](https://wiki.openjdk.org/display/SKARA/Backports)
 :::
 
-Development of the latest version of the JDK often results in bug fixes that might be interesting to include in some of the JDK update releases still being maintained. Moving a fix from a more recent release train (e.g. JDK 17) to an older release train (e.g. JDK 11) is called *backporting*.
+Development of the latest version of the JDK often results in bug fixes that might be interesting to include in some of the JDK update releases still being maintained or in a [release stabilization repository](https://openjdk.org/guide/#backporting-to-release-stabilization-fork). Moving a fix from a more recent release train (e.g. JDK 17) to an older release train (e.g. JDK 11) is called *backporting*.
 
-The guideline for what to backport into a specific update release will vary over the lifetime of that release. Initially more fixes are expected to be backported as new features and large changes introduced in a mainline release stabilize. Over time the focus will shift from stabilization to keeping it stable - the release will go into maintenance mode. This means that bug fixes that require larger disruptive changes are more likely to be made in mainline and backported to more recent release trains only, and not to older release trains.
+The guideline for what to backport into a specific release will vary over the lifetime of that release. Initially more fixes are expected to be backported as new features and large changes introduced in a mainline release stabilize. Over time the focus will shift from stabilization to keeping it stable - the release will go into maintenance mode. This means that bug fixes that require larger disruptive changes are more likely to be made in mainline and backported to more recent release trains only, and not to older release trains. Release stabilization repositories (e.g. [jdk20](https://git.openjdk.org/jdk20)) follows the process described in [JEP 3](https://openjdk.org/jeps/3) for deciding the bug fixes that should be backported.
```

Comment:
JEP 3 only sets the limits of what may and may not be included during rampdown, it doesn't give any hints of how to decide if a particular bugfix should be included. The developer of a fix will still need to decide if the change should go into the stabilization repo or not.